src/app/router/routes.ts

```diff
@@ -1,8 +1,21 @@
+import { t } from "i18next";
+
 const HOME_ROUTE = "/";
 const POOLS_ROUTE = "pools";
 const ADD_LIQUIDITY = "add-liquidity";
 const SWAP_ROUTE = "swap";
 const POOLS_ADD_LIQUIDITY = "/pools/add-liquidity";
 const POOLS_PAGE = "/pools";
 
-export { HOME_ROUTE, POOLS_ROUTE, SWAP_ROUTE, ADD_LIQUIDITY, POOLS_ADD_LIQUIDITY, POOLS_PAGE };
+const SEO_ROUTES = {
+  [`${POOLS_ROUTE}`]: {
```

Comment:
You can put just [POOLS_ROUTE], without `.

Reply:
it breaks now that i removed this, i am fixing it now